integrations/spring-boot3/src/main/java/org/eclipse/store/integrations/spring/boot/types/configuration/EclipseStoreProperties.java

```diff
@@ -202,6 +202,20 @@ public class EclipseStoreProperties
      */
     private boolean registerJdk8Handlers;
 
+    /**
+     * This flag determines if a default foundation should be automatically created.
+     * If no user-defined bean with EmbeddedStorageFoundation exists, a default one will be created.
+     * To prevent this automatic creation, set this flag to false.
+     */
+    private boolean autoCreateDefaultFoundation = true;
+
+    /**
+     * This flag determines if a default storage should be automatically created.
+     * If no user-defined bean with EmbeddedStorage exists, a default one will be created.
+     * To prevent this automatic creation, set this flag to false.
+     */
+    private boolean isAutoCreateDefaultStorage = true;
```

Comment:
Maybe rename to `autoCreateDefaultStorage`, for naming consistency (`autoCreateDefaultFoundation`)

Reply:
Ok, fixed.